### PR TITLE
JBIDE-25026 - Create N&N for Fuse Tools 10.1.0.AM3 in JBoss Tools

### PR DESCRIPTION
**Pull request by apupier**
4.5.1.AM3

Signed-off-by: Aurélien Pupier <apupier@redhat.com>

### CODE REVIEW

**Comment by apupier** [comment]
@bfitzpat Can you review it please? Have I catch the whole point?

File: documentation/whatsnew/fusetools/fusetools-news-10.1.0.AM3.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+:page-product_id: jbt_core
+:page-product_version: 4.5.1.AM3
+
+== Global Beans: improve support for Bean references
+
+It is now possible to set Bean references from User Interface when creating a new Bean:
+
+image::./images/factoryBeanReference.png[Create Factory Bean Reference]
+
+and available from properties view for edition on already existing Bean:
```
Comment (bfitzpat) [inline documentation/whatsnew/fusetools/fusetools-news-10.1.0.AM3.adoc]: Would edit to:

"Editing Bean references is also now available on the properties view when editing an existing Bean:"

File: documentation/whatsnew/fusetools/fusetools-news-10.1.0.AM3.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+image::./images/factoryBeanReference-propertiesView.png[Edit Factory Bean Reference]
+
+You can notice that validation has been added to guide users to avoid mixing Bean defined with a class name and Bean defined with a reference to another Bean.
```
Comment (bfitzpat) [inline documentation/whatsnew/fusetools/fusetools-news-10.1.0.AM3.adoc]: Would edit to:
"Additional validation has been added to help users avoid mixing Beans defined with class names and Beans defined referencing other beans."

**Comment by apupier** [comment]
@jeffmaury can you review/merge please? I don't have commit right.

**Pull request merged**